src/Features/Core/Portable/CodeRefactorings/AddMissingImports/AbstractAddMissingImportsRefactoringProvider.cs

```diff
@@ -36,7 +37,9 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             // Check pasted text span for missing imports
             var addMissingImportsService = document.GetLanguageService<IAddMissingImportsFeatureService>();
 
-            var analysis = await addMissingImportsService.AnalyzeAsync(document, textSpan, cancellationToken).ConfigureAwait(false);
+            var options = new AddMissingImportsOptions(context.Options.HideAdvancedMembers);
```

Comment:
i believe this is correct, but i'm curious what made you pass 'false' implicitly here for 'searchreferenceassemblies'?

Reply:
n/m.  different options.  ignore me.